cartoframes/keys.py

```diff
@@ -0,0 +1,56 @@
+"""API key utility functions
+"""
+import os
+import six
+import warnings
+
+if not six.PY3:
+    FileNotFoundError = OSError
```

Comment:
Cool, thanks for the info! To avoid redefining an Error, I chose to go one level up and use the more general `EnvironmentError` which means a little less compatibility code. Got my info from here: http://stackoverflow.com/questions/21367320/searching-for-equivalent-of-filenotfounderror-in-python-2?answertab=active#tab-top